frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -579,6 +581,8 @@
 
         click-color
         (fn [event]
+          (when on-select
+            ((on-select (:id color)) event))
```

Comment:
this is very strange. Why don't use a callback with two arguments? `on-select (:id color) event`

Reply:
I added this to an existing function. I think this will change when I make the seleccion global to all asset types.